jetson/percep/pcl.hpp

```diff
@@ -7,20 +7,33 @@
 /* --- Compare Line Class --- */
 //Functor that indicates where a point is in
 //relation to a line in 2D space
+//recieves angle off y axis
 class compareLine {
 public:
+    int xIntercept;
     double m;
```

Comment:
What about a similar renaming for `m` as you did for `b`?

Reply:
fs